src/i18n/translate/vi/login.json

```diff
@@ -0,0 +1,10 @@
+{
+  "enter_email": "Nhập email vào",
+  "enter_password": "Nhập mật khẩu vào",
+  "register_now": "đăng ký ngay",
+  "field_required": "Trường này là bắt buộc",
+  "right_email": "Cần nhập đúng định dạng email",
+  "max_length": "Số ký tự tối đa là: ",
```

Comment:
translate có thể truyền giá trị vào
https://stackoverflow.com/questions/39370270/how-to-pass-arguments-in-i18n-translate